mmv1/third_party/terraform/services/logging/resource_logging_project_sink_test.go

```diff
@@ -415,6 +448,66 @@ resource "google_storage_bucket" "log-bucket" {
 `, name, envvar.GetTestProjectFromEnv(), bucketName)
 }
 
+func testAccLoggingProjectSink_customWriter(name, bucketName string, serviceAccount string) string {
+	return fmt.Sprintf(`
+resource "google_service_account" "test-account1" {
```

Comment:
Add google_project datasource to get test project

```suggestion
data "google_project" "project" {}

resource "google_service_account" "test-account1" {
```

Reply:
If you need a separate project as well then you will need to create the separate project in this test and reference it as needed. For example:

```
resource "google_project" "other_project" {
  name       = "My Project"
  project_id = "%s" // should have tf-test prefix and random suffix
}
...
"${google_project.other_project.project_id}"
```

Reply:
Fixed.